src/theme.ts

```diff
@@ -4,7 +4,7 @@ const defaultTheme = createTheme();
 
 // https://www.cru.org/brand/color/
 const cruColors = {
-  yellow: '#FFCF07',
+  yellow: '#F5BE19',
```

Comment:
Is there a reason we're moving away from the official Cru yellow? https://www.cru.org/brand/color/ The old MPDX may have been using the older yellow.

Reply:
Ah, I see; I can revert it back; it does make it a little harder to view the text but not that much.

Reply:
You're right that we need to make sure it's legible. The style guide also says:

```
Take care to preserve legibility when using yellow. As a rule, avoid using yellow text on white and white text on yellow.
```

I haven't looked at the UI yet; I'm about to test it. But we should probably find an alternative if we're doing yellow-on-white or vice-versa.